rush.json

```diff
@@ -1,6 +1,6 @@
 {
   "$schema": "https://developer.microsoft.com/json-schemas/rush/v5/rush.schema.json",
-  "rushVersion": "5.112.2",
+  "rushVersion": "5.113.4",
```

Comment:
I noticed the `breaking-change` label is set on this PR. 
Why is this necessary?

Reply:
You're right it's not necessary. Removed.